### DISCUSSION TITLE
Placeholder Money/Currency Object

### DISCUSSION THREAD

**Discussion by jasper-one** [EveronLife/EveronLife#137] [Ideas]
I think we should implement a very simple Money/Currency object such as a $1 US Bill to provide us with a basic currency for now for testing purposes and also in anticipation of #81.

If this could be integrated with #134 and/or #8  it could also be the start of the currency system.

If it's a good idea, I can implement this using the conventions immediately. I don't see this being any more than a few kilobytes.

**Comment by Arkensor** [comment]
I am taking care of this as soon as #81 is all done. I have linked a few models in #9 and there was already some discussion about this there. Persistence -> Money/Quantity -> Trader and then the first milestone is done and other systems can start using those aspects. I would not add a placeholder just for a week or so now.
